libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -497,6 +500,12 @@ void AC_AttitudeControl::input_rate_bf_roll_pitch_yaw(float roll_rate_bf_cds, fl
 
 // Command an angular velocity with angular velocity smoothing using rate loops only with no attitude loop stabilization
 void AC_AttitudeControl::input_rate_bf_roll_pitch_yaw_2(float roll_rate_bf_cds, float pitch_rate_bf_cds, float yaw_rate_bf_cds)
+{
+    _ang_vel_body = input_rate_bf_roll_pitch_yaw_4(roll_rate_bf_cds, pitch_rate_bf_cds, yaw_rate_bf_cds);
+}
+
+// Command an angular velocity with angular velocity smoothing using rate loops only with no attitude loop stabilization
+Vector3f AC_AttitudeControl::input_rate_bf_roll_pitch_yaw_4(float roll_rate_bf_cds, float pitch_rate_bf_cds, float yaw_rate_bf_cds)
```

Comment:
I don't understand what making this a wrapper is doing for us?

Reply:
Its also called from input_rate_step_bf_roll_pitch_yaw() which is now required for autotune